packages/create-sitecore-jss/src/templates/nextjs-multisite/src/lib/middleware/plugins/multisite.ts

```diff
@@ -26,6 +26,10 @@ class MultisitePlugin implements MiddlewarePlugin {
       excludeRoute: () => false,
       // This function resolves site based on hostname
       getSite: siteResolver.getByHost,
+      // This function allows resolving site from cookies, which could be useful in case of Vercel preview URLs. Accepts NextRequest.
+      useCookieResolution: () => {
+        return process.env.VERCEL_ENV === 'preview';
+      },
```

Comment:
Can simplify
```suggestion
      // This function allows resolving site from sc_site cookie, which could be useful in case of Vercel preview URLs. Accepts NextRequest.
     useCookieResolution: () => process.env.VERCEL_ENV === 'preview',
```